Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationXaml.bind

```diff
@@ -289,7 +292,10 @@
                                     BorderThickness="0"
                                     ShowDismissButton="False"
                                     VerticalAlignment="Top"
-                                    RenderTransformOrigin="0.5,0">
+                                    RenderTransformOrigin="0.5,0"
+                                    AnimationDuration="@[AnimationDuration:String:100]"
```

Comment:
If you reuse the same property in the file later you can just use its name `@[AnimationDuration]` it'll grab the settings from the first one.